test/system/252-quadlet.bats

```diff
@@ -752,8 +752,32 @@ EOF
 }
 
 @test "quadlet - volume path using specifier" {
-    local tmp_path=$(mktemp -d --tmpdir=$PODMAN_TMPDIR quadlet.volume.XXXXXX)
-    local tmp_dir=${tmp_path#/tmp/}
+    # Step 1: determine what systemd is using for %T. There does not
+    # seem to be any systemctly way to find this.
```

Comment:
I think we could try `systemd-run [--user] -q --pipe printenv TMPDIR`
based on `systemd.unit(5)` `%T` defaults to $TMPDIR and if unset /tmp. Not much better but avoids the tmp unit.

Reply:
I'm not sure that's safe enough for me. From `systemd.unit(5)`, `%T` is:
> either /tmp or the path "$TMPDIR", "$TEMP" or "$TMP" are set to.